doc/source/myrocks/variables.rst

```diff
@@ -2388,6 +2388,11 @@ Specifies the timeout in seconds before deleting archived WAL files.
 Default is ``0`` (archived WAL files are never deleted).
```

Comment:
This should probably say something like "(WAL files are NOT archived)" instead.

Reply:
Good point. Fixed.